spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'User Show', type: :feature do
+  describe 'User' do
+    before(:each) do
+      @user1 = User.create! name: 'Shahier', photo: 'avatar.png', password: '123456789', email: 'shahiern@yahoo.com',
+                            bio: 'test bio', confirmed_at: Time.now
+      visit new_user_session_path
+      fill_in 'Email', with: 'shahiern@yahoo.com'
+      fill_in 'Password', with: '123456789'
+      click_button 'Log in'
+
+      @post1 = @user1.posts.create!(title: 'test title 1', text: 'test text 1', comments_counter: 0, likes_counter: 0)
+      @post2 = @user1.posts.create!(title: 'test title 2', text: 'test text 2', comments_counter: 0, likes_counter: 0)
+      @post3 = @user1.posts.create!(title: 'test title 3', text: 'test text 3', comments_counter: 0, likes_counter: 0)
+      click_on 'Shahier'
+    end
+
+    it 'shows the username of the user' do
+      expect(page).to have_content('Shahier')
+    end
+
+    it 'shows the photo for ther user' do
+      image = page.all('img')
+      expect(image.size).to eql(1)
+    end
+
+    it 'shows number of posts for the user' do
+      expect(page).to have_content('Number of posts: 3')
+    end
+
+    it 'shows the bio for the user' do
+      expect(page).to have_content('test text 1')
```

Comment:
- [ ] **[OPTIONAL]** I am adding this as optional because I tested it and it worked. So kindly add the correct content for the bio. It should be ` expect(page).to have_content('test bio')` instead of `expect(page).to have_content('test text 1')`. Also kindly remove all the files that you do not need in the spec folder because instead of having commented codes in them I suggest you just delete them to make your codebase clean.